src/genn/genn/code_generator/codeGenUtils.cc

```diff
@@ -486,4 +486,33 @@ void checkUnreplacedVariables(const std::string &code, const std::string &codeNa
         throw std::runtime_error("The "+vars+"undefined in code "+codeName+".");
     }
 }
+
+//--------------------------------------------------------------------------
+/*! \brief This function substitutes function names in a code with namespace as prefix of the function name for backends that do not support namespaces by checking that the function indeed exists in the support code and returns the substituted code.
+ */
+ //--------------------------------------------------------------------------
+std::string substituteNamespaceFunction(const std::string supportCode, const std::string code, std::string namespaceName) {
+    // Regex for function call
+    std::regex funcCallRegex("\\w+(?=\\(.*\\))");
```

Comment:
In C++ 11 you can make these slightly nicer by using 'raw' strings i.e. 
```c++
std::regex funcCallRegex(R"\w+(?=\(.*\))");
```
Also some comments explaining that the regexes do would be nice as they can be tricky to decode 😄 (e.g. I'm struggling to remember how forward lookeahead ``?=`` means)